src/main/java/randoop/main/Minimize.java

```diff
@@ -0,0 +1,1078 @@
+
+        // Simplify statement by removing the left hand side.
+        replacements.add(removeLeftHandSideSimplifiation(vdExpr));
+      }
+    }
+    return replacements;
+  }
+
+  /**
+   * If {@code currStmt} is a statement that is a passing assertion, store the value associated with
+   * the variable in the {@code primitiveValues} map.
+   *
+   * @param currStmt a statement
+   * @param primitiveValues a map of variable names to variable values, modified if {@code currStmt}
+   *     is a passing assertion, asserting a variable's value.
+   */
+  private static void storeValueFromAssertion(
+      Statement currStmt, Map<String, String> primitiveValues) {
+    // Check if the statement is an assertion regarding a value that can be used
+    // in a simplification later on.
+    if (currStmt instanceof ExpressionStmt) {
+      Expression exp = ((ExpressionStmt) currStmt).getExpression();
+      if (exp instanceof MethodCallExpr) {
+        MethodCallExpr mCall = (MethodCallExpr) exp;
+        // Check if the method call is an assertTrue statement
+        if (mCall.getName().equals("assertTrue")) {
+          // Is an assertTrue statement.
+          List<Expression> mArgs = mCall.getArgs();
+          if (mArgs.size() == 1) {
+            // Retrieve and store the value associated with the variable in the assertion.
+            Expression mExp = mArgs.get(0);
+            List<Node> children = mExp.getChildrenNodes();
+            String var = children.get(0).toString();
+            String val = children.get(1).toString();
+            primitiveValues.put(var, val);
+          }
+        }
+      }
+    }
+  }
+
+  /**
+   * Return a variable declaration statement that simplifies the right hand side to 0, false, or
+   * null, whichever is type correct. The variable declaration statement is assumed to declare only
+   * one variable, for instance, {@code int i;}. Multiple variable declarations in a single
+   * statement in the form {@code int i, j, k;} are not valid.
+   *
+   * @param vdExpr variable declaration expression representing the current statement to simplify
+   * @return a {@code Statement} object representing the simplified variable declaration expression.
+   *     Returns {@code null} if more than one variable is declared in the {@code
+   *     VariableDeclarationExpr}.
+   */
+  private static Statement rhsAssignZeroValue(VariableDeclarationExpr vdExpr) {
+    // Copy the variable declaration expression.
+    VariableDeclarationExpr exprCopy = (VariableDeclarationExpr) vdExpr.clone();
+
+    // Obtain a reference to the variable declaration.
+    List<VariableDeclarator> vars = exprCopy.getVars();
+    if (vars.size() > 1) {
+      // More than one variable declared in this expression.
+      return null;
+    }
+    VariableDeclarator vd = vars.get(0);
+
+    // Based on the declared variable type, assign the variable 0, false, or null.
+    Type type = exprCopy.getType();
+    if (type instanceof PrimitiveType) {
```

Comment:
`rhsAssignZeroValue` and `rhsAssignValueFromPassingAssertion` have very similar bodies, here and elsewhere.  Abstract out the similar code into a method that both can call, which will make Minimize.java smaller overall.